client/my-sites/plans/current-plan/current-plan-thank-you/backup-thank-you.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * External dependencies
+ */
+import { localize } from 'i18n-calypso';
+import React, { Fragment } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import ThankYou from './thank-you';
+
+const BackupProductThankYou = ( { translate } ) => (
+	<ThankYou
+		illustration="/calypso/images/illustrations/security.svg"
+		showCalypsoIntro
+		showContinueButton
+		title={ translate( 'Hello backups!' ) }
+	>
+		<Fragment>
+			<p>{ translate( 'We just finished settup up backups for you.' ) }</p>
```

Comment:
Did you mean `setting up`?

```suggestion
			<p>{ translate( 'We just finished setting up backups for you.' ) }</p>
```